spec/codeowners/checker_spec.rb

```diff
@@ -439,4 +439,20 @@ def setup_git_for_project
       expect(subject.main_group).to be_a(Codeowners::Checker::Group)
     end
   end
+
+  describe '#list_files_for_owner' do
+    subject { described_class.new folder_name, from, to }
+
+    it 'does stuff' do
```

Comment:
```suggestion
    it 'filter files by owner' do
```

Reply:
Doh!  Thought I updated that - nice catch!